.storybook/stories/illustrations.stories.js

```diff
@@ -24,6 +24,29 @@ export default {
   title: addStoryInGroup(FOUNDATION, 'Illustrations'),
 };
 
+export const _24X24 = () => (
+  <Box style={gridStyles}>
+    {Object.keys(Illustrations).map((key, index) => {
+      if (key.includes('24X24')) {
+        const IllustrationToRender = Illustrations[key];
+
+        return (
+          <Box key={index} style={itemStyles} {...itemProps}>
```

Comment:
```suggestion
    {Object.keys(Illustrations).map(key => {
      if (key.includes('24X24')) {
        const IllustrationToRender = Illustrations[key];

        return (
          <Box key={key} style={itemStyles} {...itemProps}>
```
I think key is always unique anyways right

Reply:
Yep, you're right. Using key as the key makes more sense. I'll change this!